zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -73,18 +72,10 @@ object DbReader {
     override def getk8sNodepoolsToDeleteCandidate: Stream[F, NodepoolToScan] =
       activeNodepoolsQuery.stream.transact(xa)
 
-    override def markNodepoolAndAppStatusDeleted(nodepoolId: Long): F[Unit] = {
+    override def updateNodepoolAndAppStatus(nodepoolId: Long, status: String): F[Unit] = {
```

Comment:
Should we also use a `Status` ADT the way we do in other codebases for more type safety?

Reply:
I feel ok without an ADT for now. If we publish leo models, they won't be needed here.